domains/kaz.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Add some description",
+  "repo": "https://github.com/kazz0001/kazz0001.github.io",
+  "owner": {
+    "username": "kazz0001",
+    "email": "badchicken11690@email",
```

Comment:
invalid email....

Reply:
if you dont wanna leak email then set email as "" and add any other social media such as twitter/reddit/discord

Reply:
oh i'm so sorry, here badchicken11690@gmail.com

Reply:
...